### PR TITLE
Update react-resources.md

### PR DESCRIPTION
**Pull request by AdebayoIbrahim**
- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

Updated react-resources.md to fit beginners guide

### CODE REVIEW

File: pages/react-resources.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
...
 | 3. CSS Grid CheatSheet by CSS-Tricks | [![Web Development](https://ik.imagekit.io/gaba/css-tricks-e1394062740966.png?updatedAt=1678565916889)](https://css-tricks.com/snippets/css/complete-guide-grid/)| [Here you go](https://css-tricks.com/snippets/css/complete-guide-grid/)
-| 4. React Js Testing by Codevolution | [![Web Development](https://i.ibb.co/ryn3Qck/jest.png "Web Development")](https://www.youtube.com/watch?v=T2sv8jXoP4s&list=PLC3y8-rFHvwirqe1KHFCHJ0RqNuN61SJd)| [Here you go](https://www.youtube.com/watch?v=T2sv8jXoP4s&list=PLC3y8-rFHvwirqe1KHFCHJ0RqNuN61SJd) 
\ No newline at end of file
+| 4. React Js Testing by Codevolution | [![Web Development](https://i.ibb.co/ryn3Qck/jest.png "Web Development")](https://www.youtube.com/watch?v=T2sv8jXoP4s&list=PLC3y8-rFHvwirqe1KHFCHJ0RqNuN61SJd)| [Here you go](https://www.youtube.com/watch?v=T2sv8jXoP4s&list=PLC3y8-rFHvwirqe1KHFCHJ0RqNuN61SJd) 
+| 7. Learn React Basics | [![Web Development](https://i.ibb.co/pbf14TL/w3-pull-res.png "WebDevelopment")](https://www.w3schools.com/react)| [Here you go](https://www.w3schools.com/react)
```
Comment (hasnainmakada-99) [inline pages/react-resources.md]: Fix the numbering here @AdebayoIbrahim

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]
Please look onto the requested changes @AdebayoIbrahim

**Comment by AdebayoIbrahim** [comment]
> Please look onto the requested changes @AdebayoIbrahim 

Done boss 🙌

**Review by hasnainmakada-99 — APPROVED** [review]
LGTM @AdebayoIbrahim

**Comment by hasnainmakada-99** [comment]
@all-contributors please add @AdebayoIbrahim as a contributor of content

**Comment by hasnainmakada-99** [comment]
@AdebayoIbrahim I am adding you as a contributor to the OSWH project as you have provided a lot of useful resources to the community, once again thanks for all of your contributions

**Comment by AdebayoIbrahim** [comment]
> @AdebayoIbrahim I am adding you as a contributor to the OSWH project as you have provided a lot of useful resources to the community, once again thanks for all of your contributions

@hasnainmakada-99  Wow Thanks I am delighted to hear that Boss

**Pull request merged**